src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -2349,21 +2340,7 @@ protected function SetDeleteBlackList($sTableId = false, $sRecordId = false)
      */
     public function IsInDeleteBlackList($sTableId = false, $sRecordId = false)
```

Comment:
As we now know that this always returns false we should also remove the call in IsAllowedDelete().